todo_app/templates/index.html

```diff
@@ -6,12 +6,24 @@
     <h1 class="display-4">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
   </div>
-
+  <div>
+    <form action="" method="post" class="form-example">
+      <div class="form-example">
+        <label for="field_name">Enter task: </label>
+        <input type="text" name="field_name" id="field_name">
```

Comment:
You could add some client side validation here:
```suggestion
        <input type="text" name="field_name" id="field_name" required>
```